### DISCUSSION TITLE
Remap "a" to open annotation editor window

### DISCUSSION THREAD

**Discussion by xmfbraga** [sumatrapdfreader/sumatrapdf#3783] [Q&A]
Hi, everybody. 

Thanks for this great program!

I frequently use Sumatra to read my student's text and, generally, I need to insert some comments. I, usually, highlight the original text and I insert my suggestions. However, it takes much time to press Shift+a (capital A) to highlight the text and open annotation editor window. 

Is there a way to make key "a" (small a) highlight text and also open annotation editor window?

Thanks in advance.

**Comment by GitHubRulesOK** [comment]
@kjk 
CmdCreateAnnotHighlight seems to be missing the alternate "shifted" command

**Reply by GitHubRulesOK** [reply]
@xmfbraga 
I have raised as an issue see https://github.com/sumatrapdfreader/sumatrapdf/issues/3784
